dbt/profiles.yml

```
@@ -3,11 +3,19 @@ dbt-civis:
   default: redshift_adapter
   outputs:
     redshift_adapter:
-      type: redshift
+      type: "{{ env_var('DATABASE_TYPE', 'redshift') }}"
       host: "{{ env_var('DATABASE_HOST') }}"
       port: "{{ env_var('DATABASE_PORT') | as_number }}"
       user: "{{ env_var('DATABASE_USERNAME') }}"
       pass: "{{ env_var('DATABASE_PASSWORD') }}"
       dbname: "{{ env_var('DATABASE_NAME') }}"
-      schema: "{{ env_var('DATABASE_SCHEMA', 'default_schema_name') }}"
+      schema: "{{ env_var('DATABASE_SCHEMA') }}"
       threads: 4
+
+    bigquery_service_account_adapter:
+      threads: 4
+      method: "{{ env_var('GCP_AUTH_METHOD') }}"
+      type: "{{ env_var('DATABASE_TYPE') }}"
+      project: "{{ env_var('GCP_PROJECT_ID') }}"
+      schema: "{{ env_var('DBT_DATASET_NAME') }}"
+      keyfile: "{{ env_var('GCP_SERVICE_ACCOUNT_KEYFILE_JSON') }}"
```

are the `GCP_*` env variables defined by bigquery? 

I think it could be helpful to have a comment for bigquery_service_account_adapter and one for redshift_adapter explaining the different env variables/where they come from. Or do we have a readme? 

E.g.
```suggestion
    # ---
    # * GCP_AUTH_METHOD: used by x, auto-populated(defined?) by Platform's y variable?
    # * DATABASE_TYPE: ...
      method: "{{ env_var('GCP_AUTH_METHOD') }}"
      type: "{{ env_var('DATABASE_TYPE') }}"
      project: "{{ env_var('GCP_PROJECT_ID') }}"
      schema: "{{ env_var('DBT_DATASET_NAME') }}"
      keyfile: "{{ env_var('GCP_SERVICE_ACCOUNT_KEYFILE_JSON') }}"
```

---

no these are not defined by dbt or bigquery. GCP_ is just a prefix i thought would help with readability.